.circleci/config.yml

```diff
@@ -69,6 +69,19 @@ jobs:
           path: ~/junit
       - store_artifacts:
           path: ~/junit
+      - run:
+          name: Make jar directory
+          command: mkdir /tmp/artifacts
+      - run:
+          name: Move jars
+          command: cp dockstore-webservice/target/dockstore*.jar /tmp/artifacts
```

Comment:
Wonder if it is worth doing the same thing as https://github.com/dockstore/dockstore/blob/develop/Dockerfile#L27 and avoid copying the source jar